environments/adapters/ollama.py

```
@@ -43,7 +45,13 @@ def verify_required_model(
         if isinstance(models, list) and isinstance(model, dict)
     } if isinstance(models, list) else set()
     if model_name not in names:
-        raise OllamaPreparationError(f"Ollama model {model_name} is required")
+        raise OllamaPreparationError(
+            f"Ollama model {model_name} is required. Run: "
+            "sudo -u digital-souls env "
+            "HOME=/var/lib/digital-souls/home "
+            "OLLAMA_MODELS=/var/lib/digital-souls/models/ollama "
+            f"ollama pull {shlex.quote(model_name)}"
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)